resources/LayerSelector.css

```diff
@@ -48,7 +48,6 @@
   -webkit-box-shadow: 0 1px 5px rgba(0,0,0,0.4);
   box-shadow: 0 1px 5px rgba(0,0,0,0.4);
   background: #fff;
-  -webkit-border-radius: 5px;
```

Comment:
yes stylus would add this but if you want to use the css without touching stylus i would think you might need this or is the vendor prefix not necessary anymore? Should we remove all of the prefixes?